vision_datasets/image_object_detection/detection_as_kvp_dataset.py

```diff
@@ -14,26 +14,43 @@
 class DetectionAsKeyValuePairDatasetBase(VisionDataset):
     """Dataset class that access Detection datset as KeyValuePair dataset."""
 
-    def __init__(self, detection_dataset: VisionDataset):
+    def __init__(
+        self,
+        detection_dataset: VisionDataset,
+        include_class_names: bool = True,
+        custom_schema_description: Optional[str] = None,
+    ):
         """
         Initializes an instance of the DetectionAsKeyValuePairDataset class.
         Args:
             detection_dataset (VisionDataset): The detection dataset to convert to key-value pair dataset.
+            include_class_nams (bool): If True, include class names in the schema. If False, exclude class names in the schema. When class names are excluded, the task becomes open-vocabulary and the model can output any class name.
```

Comment:
Nit: Maybe remove the part about the model, since this is part of the datasets doc and may be confusing?

Reply:
updated